CHANGELOG.md

```diff
@@ -1,16 +1,32 @@
 ## Unreleased
 
+FEATURES:
+* Support Kubernetes health probe synchronization with Consul for connect injected pods via `connectInject.healthChecks` [[GH-651](https://github.com/hashicorp/consul-helm/pull/651)].
+  The default behavior for this feature is `enabled: true`.
+  See [https://www.consul.io/docs/k8s/health](https://www.consul.io/docs/k8s/health) for more information.
+  **It is recommended to enable TLS with this setting enabled.**
+
+BUG FIXES:
+* Support new versions of timeout in ent-license job [[GH-654](https://github.com/hashicorp/consul-helm/pull/654)].
+
+DEPRECATIONS:
+* `create-inject-token` in the server-acl-init job has been undeprecated.
+  `create-inject-auth-method` has been deprecated and replaced by `create-inject-token`.
+  `create-inject-namespace-token` in the server-acl-init job has been deprecated. [[GH-665](https://github.com/hashicorp/consul-helm/pull/665)].
+  See: [[GH-368](https://github.com/hashicorp/consul-k8s/pull/368)] for additional documentation regarding these changes.
```

Comment:
These are consul-k8s deprecations. I don't think they should be listed for the consul-helm changelog since there's nothing for users to do.

What we do need to list is a breaking change that only consul-k8s v0.20.0 is supported with this helm chart version.